plugins/inventory/kubevirt.py

```diff
@@ -122,15 +122,15 @@
 # File must be named kubevirt.yaml or kubevirt.yml
 
 # Authenticate with token, and return all virtual machines for all namespaces
-plugin: kubevirt
+plugin: community.kubevirt.kubevirt
```

Comment:
You (probably) need to extend `choices` of `plugin` in `DOCUMENTATION.options` above to also allow `community.kubevirt.kubevirt`. But please keep the shortname there, otherwise you might break things once the new redirects are in place ;)

Reply:
Thanks @felixfontein, does your comment apply only to this specific inventory plugin or all plugins ? Between your comment and the one from @gundalow, I'm confused about where the FQCN is needed at all.

FWIW I was going off of https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#documentation

Reply:
Maybe a difference between modules and other plugins

Reply:
This is about the `plugin` option for inventory plugins. This is not about the plugin's name in `DOCUMENTATION` or anything else :)

Reply:
(AFAIK the `plugin` field is used by Ansible's `auto` inventory plugin to identify the correct plugin to use for this YAML file. Or something like that. Never really looked into how it works in detail.)